batchflow/models/tf/mobilenet.py

```diff
@@ -112,19 +152,9 @@ def block(cls, inputs, strides=1, double_filters=False, width_factor=1, name=Non
         return conv_block(inputs, 'Cna cna', filters, [3, 1], name=name, strides=[strides, 1], **kwargs)
 
 
-_V2_DEFAULT_BODY = [
-    dict(repeats=1, filters=16, expansion_factor=1, strides=1),
-    dict(repeats=2, filters=24, expansion_factor=6, strides=2),
-    dict(repeats=3, filters=32, expansion_factor=6, strides=2),
-    dict(repeats=4, filters=64, expansion_factor=6, strides=2),
-    dict(repeats=3, filters=96, expansion_factor=6, strides=1),
-    dict(repeats=3, filters=160, expansion_factor=6, strides=2),
-    dict(repeats=1, filters=320, expansion_factor=6, strides=1),
-]
-
 class MobileNet_v2(TFModel):
-    """ MobileNet version 2
-
+    """ Base class for MobileNets after v2.
```

Comment:
```suggestion
    """ MobileNet version 2
```
There is no reference at the top of the file to the article about MobileNet v2

Reply:
there is, the name of the article didn't contain the name of the network